### PR TITLE
achievements: adding help links to popups [fixes #3433]

### LINKED ISSUE #3433: Help popup in Achievements: Link to tips
In each of the help popups, it would be great to have a link that a wiki page that explains things much more deeply.

For instance the help popup for "Featured Images" could link to https://commons.wikimedia.org/wiki/Commons:Featured_picture_candidates or https://commons.wikimedia.org/wiki/Commons:Featured_pictures

### PR DESCRIPTION
**Pull request by gerardolgvr**
**Description (required)**
achievements: adding links to popups for extra information

Fixes #3433 

What changes did you make and why?
- New string added for "Read more" string in strings.xml
- New method added for lauchAlertWithLink to open link using an intent
- Associate this new method for the existing popups that requires extra info

**Tests performed (required)**

Tested with ProdDebug on Redmi Note 7 with API level 28 and Nexus 5X API level 28 (emulator).

**Screenshots (for UI changes only)**
image

Need help? See https://support.google.com/android/answer/9075928

---

_Note: Please ensure that you have read CONTRIBUTING.md if this is your first pull request._

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/achievements/AchievementsActivity.java [hunk 1]
```diff
@@ -467,38 +467,50 @@ public void showAlert(Bitmap screenshot){
 
     @OnClick(R.id.images_upload_info)
     public void showUploadInfo(){
-        launchAlert(getResources().getString(R.string.images_uploaded)
-                ,getResources().getString(R.string.images_uploaded_explanation));
+        launchAlertWithLink(getResources().getString(R.string.images_uploaded)
+                ,getResources().getString(R.string.images_uploaded_explanation)
+                , "https://commons.wikimedia.org/wiki/Commons:Project_scope"
```
Comment (neslihanturan) [inline app/src/main/java/fr/free/nrw/commons/achievements/AchievementsActivity.java:472]: Thanks @gerardolgvr , all looks good except hardcoded urls. Can you use final variables instead?

Comment (gerardolgvr) [reply app/src/main/java/fr/free/nrw/commons/achievements/AchievementsActivity.java:472]: Sure! I'm gonna do the changes. Thanks for the review

File: app/src/main/java/fr/free/nrw/commons/achievements/AchievementsActivity.java [hunk 2]
```diff
@@ -19,6 +19,7 @@
 import android.widget.RelativeLayout;
 import android.widget.TextView;
 
+import android.widget.Toast;
```
[the remark this answers is not in the document]
Comment (luislizarraga) [reply app/src/main/java/fr/free/nrw/commons/achievements/AchievementsActivity.java]: No worries, thanks! 😄

**Comment by luislizarraga** [comment]
Where you able to add tests?

**Comment by gerardolgvr** [comment]
> Where you able to add tests?

Sorry, I thought that I replied before.  

I tried for my own try to run the existing test but I didn't have success u.u.  (The IDE showed me some errors)  I asked for help at the original issue but I'm still waiting for answers. For the moment I did manual tests at the links and work well but as soon I discover how to solve the problem or receive some help  I can add it :)

**Review by neslihanturan — CHANGES_REQUESTED** [review]
I am good to merge this as soon as conflicts are fixed :)

**Comment by neslihanturan** [comment]
Hey, this PR seems like sleeping. Feel free to reopen if needed:)

**Pull request closed without merging**